word_language_model/README.md

```diff
@@ -5,10 +5,14 @@ The trained model can then be used by the generate script to generate new text.
 
 ```bash
 python main.py --cuda --epochs 6           # Train a LSTM on Wikitext-2 with CUDA.
+python main.py --xpu --epochs 6            # Train a LSTM on Wikitext-2 with Intel XPU.
```

Comment:
Drop. These cmdlines actually don't give any additional info. It's clear that you can use something else instead of `--cuda`.